cassandra-migration/src/main/java/org/cognitor/cassandra/migration/filter/IdempotentFilter.java

```diff
@@ -0,0 +1,8 @@
+package org.cognitor.cassandra.migration.filter;
+
+public class IdempotentFilter implements ScriptFilter {
```

Comment:
Maybe name it NoOpFilter as Idempotent just means that it produces always the same result for the same input but here we are not doing anything with the input.

Reply:
done